src/components/helpers/chocolate_sauce.ts

```diff
@@ -203,8 +203,8 @@ export async function chocolateSauce(url: string, item: number = 0, starter: num
 	var article: any = {};
 	var article_title: string = '';
 	var article_body: string = '';
-	var article_image: string = '';
-	var article_logo: string = '';
+	var article_image: any = '';
```

Comment:
In what case is this failing? I'm trying to think of a reason anything besides a string (or null/empty) would end up here to start with.

Reply:
Sometimes the image pulled ends up an object. I forget why